docs/connect/erlang.md

```diff
@@ -0,0 +1,119 @@
+(connect-erlang)=
+
+# Erlang
+
+:::{div} sd-text-muted
+Connect to CrateDB from Erlang applications.
+:::
+
+## ODBC
+
+:::{rubric} About
+:::
+
+Erlang includes an [ODBC application] out of the box that provides an
+interface to communicate with relational SQL-databases, see also
+[Erlang ODBC examples].
+
+::::{todo}
+Enable this include with the [ODBC patch](https://github.com/crate/cratedb-guide/pull/411).
+```md
+:::{include} _odbc-setup-widget.md
+:::
+```
+::::
+
+:::{rubric} Synopsis
+:::
+
+Before running the example, ensure the PostgreSQL ODBC driver is
+installed on your system.
+
+`odbc_example.erl`
+```erlang
+-module(odbc_example).
+
+main(_) ->
+    odbc:start(),
+    {ok, Ref} = odbc:connect("Driver={PostgreSQL ODBC Driver(UNICODE)};Server=localhost;Port=5432;Uid=crate;Pwd=crate", []),
+    io:fwrite("~p~n", [odbc:sql_query(Ref, "SELECT mountain, height FROM sys.summits ORDER BY height DESC LIMIT 3")]),
+    init:stop().
+```
+
+Start CrateDB using Docker or Podman, then compile and invoke the example program.
+```shell
+docker run --rm --publish=5432:5432 crate -Cdiscovery.type=single-node
+```
+```shell
+escript odbc_example.erl
+```
+
+
+## epgsql
+
+[epgsql] is the designated Erlang PostgreSQL client library. 
+
+`rebar.config`
+```erlang
+{deps,
+ [
+  {epgsql, ".*", {git, "https://github.com/epgsql/epgsql.git", {tag, "4.8.0"}}}
+ ]}.
+```
+`epgsql_example.erl`
+```erlang
+-module(epgsql_example).
+-export([start/0]).
+
+start() ->
+    {ok, C} = epgsql:connect(#{
+        host => "localhost",
+        username => "crate",
+        password => "crate",
+        database => "doc",
+        port => 5432,
+        timeout => 4000
+    }),
+    {ok, _, Result} = epgsql:squery(C, "SELECT mountain, height FROM sys.summits ORDER BY height DESC LIMIT 3"),
+    io:fwrite("~p~n", [Result]),
+    ok = epgsql:close(C),
+    init:stop().
+```
+
+Start CrateDB using Docker or Podman, then compile and invoke the example program.
+```shell
+docker run --rm --publish=5432:5432 crate -Cdiscovery.type=single-node
+```
+```shell
+rebar3 compile
+erlc epgsql_example.erl
+erl -pa ebin ./_build/default/lib/epgsql/ebin ./_build/default/lib/epgsql/include -noshell -run epgsql_example
+```
```

Comment:
@coderabbitai: Is it possible to use `rebar3` to also **invoke** the `epgsql_example`?

Reply:
I haven't been able to improve the incantation due to this and the other obstable.

Reply:
It looks like this incantation works.

```shell
rebar3 compile
erlc epgsql_example.erl
rebar3 shell --eval 'epgsql_example:start().'
```

/cc @coderabbitai